ui/static/ui/js/lr_exports.jsx

```diff
@@ -0,0 +1,77 @@
+define("lr_exports",
+  ['reactaddons', 'jquery', 'lodash', 'utils'], function (React, $, _, Utils) {
+  'use strict';
+
+  var ExportsComponent = React.createClass({
+    componentDidMount: function() {
```

Comment:
I believe we do minimize our javascript using compressor when it's in production, which should strip out all comments. You can verify this on lore-ci